### PR TITLE
Create options table

### LINKED ISSUE #93: Create options table
default config data should be defined in an array.

### PR DESCRIPTION
**Pull request by bambamboole**
THis PR adds an Option model. 
It can be used to save values for different things to the database.

This is the foundation in the backend with option declaration and resolving with their values. I also prepared an endpoint to create and update options.

The next step is to introduce a frontend for this which renders the fields depending in their type.

resolves #93

### CODE REVIEW

File: tests/Feature/Backend/Options/ListOptionsTest.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+namespace Bambamboole\LaravelCms\Tests\Feature\Backend\Options;
+
+use Bambamboole\LaravelCms\Models\Option;
+use Bambamboole\LaravelCms\Tests\TestCase;
+
+class ListOptionsTest extends TestCase
+{
+    /** @test */
+    public function it_merges_the_values_to_the_fields()
```
[the remark this answers is not in the document]
Comment (slashplusweb) [reply tests/Feature/Backend/Options/ListOptionsTest.php]: maybe you'll even find a better name...

[the remark this answers is not in the document]
Comment (bambamboole) [reply tests/Feature/Backend/Options/ListOptionsTest.php]: I have put the test into a OptionFieldsResolverTest. Now it is clear what the it is. 
I like the thing with the it if it is clear by the name of the test class what it means

**Comment by bambamboole** [comment]
@slashplusweb ich habe mal auch eine simple frontend implementierung gemacht. wenn das so ok ist, dann würde ich das erstmal mergen. ansonsten kannst du es gerne noch hübscher machen :-)

**Comment by bambamboole** [comment]
Ok, I will merge this and create a new issue to make this beautiful

**Pull request merged**